src/schema/schema_create.c

```diff
@@ -433,6 +433,70 @@ __wt_find_import_metadata(WT_SESSION_IMPL *session, const char *uri, const char
     return (0);
 }
 
+/*
+ * __schema_is_tiered_storage_shared --
+ *     Check whether the table is configured for tiered storage, and if so, whether the tiered table
+ *     is shared.
+ */
+static bool
+__schema_is_tiered_storage_shared(WT_SESSION_IMPL *session, const char *config)
+{
+    WT_CONFIG_ITEM cval;
+
+    /*
+     * The tiered storage shared table needs to have two column groups that are
+     * pointed to the underlying active and shared files. The following checks are
```

Comment:
" that are pointed to" should be "that point to"

Reply:
Thanks. Fixed.